bzt/modules/python.py

```diff
@@ -52,6 +52,8 @@ class ApiritifNoseExecutor(SubprocessedExecutor):
     def __init__(self):
         super(ApiritifNoseExecutor, self).__init__()
         self._tailer = FileReader(file_opener=lambda _: None, parent_logger=self.log)
+        self._iteration_number = None
```

Comment:
Why store these fields at all? Why not just make iteration listener to receive `iteration_started(time)` and `iteration_ended(time)` so it decides itself what to store and do?

Reply:
I agree with that. Please, see the current version of the PR.